package.md

```diff
@@ -0,0 +1,38 @@
+# Package
```

Comment:
Let's move to installation.md? I think this info is better to be there.

Reply:
Ok